packages/sit-onyx/src/components/OnyxFormElement/types.ts

```diff
@@ -0,0 +1,46 @@
+import type { FormErrorMessages } from "../../composables/useCustomValidity";
+import type { RequiredMarkerProp } from "../../composables/required";
+
+export type OnyxFormElementProps = RequiredMarkerProp & {
```

Comment:
Should we also update the prop types for input/textarea to re-use this so we don't need to duplicate the descriptions? Somehting like

Pick<OnyxFormElementProps, "label" | "hideLabel"> etc.

Reply:
I also thought this might be nice but wasn't sure :) I'll do it where the components don't have additional docs 👍